druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/MahaLookupOperatorConversion.java

```diff
@@ -30,7 +31,7 @@
 
     private static final SqlFunction SQL_FUNCTION = OperatorConversions
             .operatorBuilder(DRUID_FUNC_NAME)
-            .operandTypes(SqlTypeFamily.ANY, SqlTypeFamily.CHARACTER, SqlTypeFamily.CHARACTER, SqlTypeFamily.CHARACTER)
+            .operandTypes(SqlTypeFamily.ANY, SqlTypeFamily.CHARACTER, SqlTypeFamily.CHARACTER, SqlTypeFamily.CHARACTER, SqlTypeFamily.CHARACTER, SqlTypeFamily.CHARACTER)
```

Comment:
secondaryColOverrideMap is commonly used feature, we can move it right after missing value and push KeyOverrideMap  to last.

Reply:
You only want it moved around inside the lookup ordering itself, but retain the same ordering on all other existing lookups (non-sql), correct?